README.md

```diff
@@ -122,6 +123,23 @@ $ ocic pod create --config test/testdata/sandbox_config.json
 ```
 # ocic ctr list
 ```
+#### Recovery from errors
```

Comment:
mmm do we still have this issue after the work done on sandboxes/containers restore?